### PR TITLE
add azure st support

### PR DESCRIPTION
**Pull request by mirnawong1**
Merge: Dec 15th or 16th

this pr clarifies that azure ST is supported for SL

- added to the tenancy table
- added to the prerequs

[    ] needs PM review
[ x   ]  needs docs review

[docs notion}(https://www.notion.so/dbtlabs/SL-support-in-Azure-ST-15abb38ebda780ce8ed1dd34843bf413?pvs=4)

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-sl-azure-st-dbt-labs.vercel.app/docs/dbt-versions/release-notes

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## December 2024
 
+- **New**: The dbt Semantic Layer now supports Azure Single-tenant deployments. Refer to [Set up the dbt Semantic Layer](/docs/use-dbt-semantic-layer/setup-sll) for more information on how to get started.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes.md]:
```suggestion
- **New**: The dbt Semantic Layer now supports Azure Single-tenant deployments. Refer to [Set up the dbt Semantic Layer](/docs/use-dbt-semantic-layer/setup-sl) for more information on how to get started.
```

File: website/snippets/_v2-sl-prerequisites.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
-- Have a dbt Cloud Team or Enterprise account. Single-tenant accounts should contact their account representative for setup.
-- Ensure your production and development environments use [dbt version 1.6 or higher](/docs/dbt-versions/upgrade-dbt-version-in-cloud).
+- Have a dbt Cloud Team or Enterprise account.
+   - Available on all [tenants](/docs/cloud/about-cloud/tenancy). Single-tenant accounts should contact their account representative for setup.
```
Comment (mirnawong1) [inline website/snippets/_v2-sl-prerequisites.md]:
```suggestion
   - Available on all [tenant configurations](/docs/cloud/about-cloud/tenancy). Single-tenant accounts should contact their account representative for setup.
```

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

Thanks for creating this PR. I've approved this for you.

Kind Regards
Natalie

File: website/snippets/_v2-sl-prerequisites.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
-- Have a dbt Cloud Team or Enterprise account. Single-tenant accounts should contact their account representative for setup.
-- Ensure your production and development environments use [dbt version 1.6 or higher](/docs/dbt-versions/upgrade-dbt-version-in-cloud).
+- Have a dbt Cloud Team or Enterprise account.
+   - Available on all [tenant configurations](/docs/cloud/about-cloud/tenancy). Single-tenant accounts should contact their account representative for setup.
```
Comment (runleonarun) [inline website/snippets/_v2-sl-prerequisites.md]:
```suggestion
   - Available on all [tenant configurations](/docs/cloud/about-cloud/tenancy). Single-tenant accounts should contact your account representative for setup.
```

**Pull request merged**